README.md

```diff
@@ -48,11 +48,10 @@ For F# Compiler on OSX and Linux (see .travis.yml for build steps)
 
 For Visual F# IDE Tools 4.1 development (Windows)
 
-- [Visual Studio 15](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs.aspx)
-  - Select Visual C++ / Common Tools For Visual C++
-  - Visual F#
-  - Visual Studio Extensibility Tools
-- [Visual Studio 15 SDK](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs.aspx)
+- [Visual Studio 2017 RC](https://www.visualstudio.com/vs/visual-studio-2017-rc/#downloadvs)
+  - Under the "Windows" workloads, select ".NET desktop development".
+    - Select "F# language suport" under the optional components.
```

Comment:
In RC1 F# was checked by default in ".NET desktop development" group. I cannot install RC2 since the installer does not work behind the proxy, so I cannot check if it's changed.